src/bin/cli.ts

```diff
@@ -31,7 +31,7 @@ const writeGeneratedFile = (
   writeFileSync(
     filepath,
     options.format
-      ? format(content, { parser: getPrettierParser(filepath) })
+      ? await format(content, { parser: getPrettierParser(filepath) })
```

Comment:
In the upgraded version of `prettier`, format returns a promise, so now has to be awaited.